Issues/Week377.md

```diff
@@ -7,19 +7,19 @@
 
 **Tools/Controls**
 
-* 
+*
 
 **Business/Career**
 
-* 
+*
 
 **UI/UX**
 
-* 
+*
 
 **Videos**
 
-* 
+* [Create a placeholder loading screen with SwiftUI Redacted (SwiftUI Tutorial, Xcode 12, SwiftUI 2.0)](https://youtu.be/cfwEt__pnvA), by [@tundsdev](https://twitter.com/tundsdev)
 
 **Credits**
 
```

Comment:
Thx @tunds! Pls add yourself to the credits section after zntfdr

Reply:
@dkhamsing Thanks I will do, can this be a link to my twitter?

Reply:
@dkhamsing This has been done now, thanks for the heads up